host_core/lib/host_core/actors/actor_module.ex

```diff
@@ -690,7 +690,7 @@ defmodule HostCore.Actors.ActorModule do
       }
       |> CloudEvent.new(evt_type)
 
-    topic = "wasmbus.evt.#{prefix}"
```

Comment:
Should we make `wasmbus.evt` and `wasmbus.rpcevt` constant prefixes somewhere?

Reply:
eventually, yes... but I've got some thoughts for how to centralize publishing of events during the "big refactor" that we want to do.

Reply:
swapped to use module constants